modules/virt-defining-storageclass.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-preparing-cdi-scratch-space.adoc
+
+[id="virt-defining-storageclass_{context}"]
+= Defining a storage class
+
+Dynamically provision scratch space for CDI operations by defining a storage class in the `HyperConverged` custom resource (CR).
```

Comment:
I think we should change the wording here.  The configuration references an existing storage class which CDI will use when allocating scratch space for certain operations.